collect_app/src/main/res/xml/aggregate_preferences.xml

```diff
@@ -19,5 +19,23 @@
             android:key="password"
             android:title="@string/password" />
     </PreferenceCategory>
-
+    <PreferenceCategory
+        android:key="@string/submission_transport"
```

Comment:
This key looks as if it's unused and btw it's rather not good to use translatable strings for keys right?

Reply:
Yes you are right.